set.mm

```diff
@@ -837767,6 +838014,1624 @@ have GLB (expanded version).  (Contributed by Zhi Wang,
+      fucoco2.z $e |- ( ph -> Z e. W ) $.
+      fucoco2.j $e |- J = ( Hom ` T ) $.
+      fucoco2.a $e |- ( ph -> A e. ( X J Y ) ) $.
+      fucoco2.b $e |- ( ph -> B e. ( Y J Z ) ) $.
+      $( Composition in the source category is mapped to composition in the
+         target.  See also ~ fucoco .  (Contributed by Zhi Wang,
+         3-Oct-2025.) $)
+      fucoco2 $p |-
+  ( ph -> ( ( X P Z ) ` ( B ( <. X , Y >. .x. Z ) A ) )
+                         = ( ( ( Y P Z ) ` B )
+                             ( <. ( O ` X ) , ( O ` Y ) >. .xb ( O ` Z ) )
+                             ( ( X P Y ) ` A ) ) ) $=
+        ( c1st cfv c2nd cnat co cxp cfunc xpcfucbas eleqtrd xpcfuchom xp1st syl
+        wcel xp2nd cop wceq 1st2nd2 fucoco ) ABCDEFGBUJUKZBULUKZHIJCUJUKZKOUJUK
+        ZOULUKZPUJUKZPULUKZQUJUKZQULUKZMCULUKZOPQABVKVMEKUMUNZUNZVLVNDEUMUNZUNZ
+        UOZVBZVHVSVBABOPLUNWBUHAEKUPUNZDEUPUNZUOZEKDIELOPREKDIERUQZUGAONWFUDUCU
+        RZAPNWFUEUCURZUSURZBVSWAUTVAAWCVIWAVBWJBVSWAVCVAACVMVOVRUNZVNVPVTUNZUOZ
+        VBZVJWKVBACPQLUNWMUIAWFEKDIELPQRWGUGWIAQNWFUFUCURZUSURZCWKWLUTVAAWNVQWL
+        VBWPCWKWLVCVATAOWFVBOVKVLVDVEWHOWDWEVFVAAPWFVBPVMVNVDVEWIPWDWEVFVAAQWFV
+        BQVOVPVDVEWOQWDWEVFVAAWCBVHVIVDVEWJBVSWAVFVAAWNCVJVQVDVEWPCWKWLVFVASUBR
+        UAVG $.
+    $}
+
+    $d C m n x y z $.  $d D m n x y z $.  $d E m n x y z $.  $d O m n x y z $.
+    $d P m n x y z $.  $d Q m n x y z $.  $d T m n x y z $.  $d m n ph x y z $.
+    fucofunc.c $e |- ( ph -> C e. Cat ) $.
+    fucofunc.d $e |- ( ph -> D e. Cat ) $.
+    fucofunc.e $e |- ( ph -> E e. Cat ) $.
+    $( The functor composition bifunctor is a functor.
+
+       However, it is unlikely the unique functor compatible with the functor
+       composition.  As an counterexample, let ` C ` and ` D ` be categories of
+       one object and one morphism, for example, ` ( SetCat `` 1o ) ` , and
+       ` E ` be a category with two objects equipped with only two non-identity
+       morphisms ` f ` and ` g ` , pointing in the same direction.  It seems
+       possible to map the ordered pair of natural transformations
+       ` <. a , i >. ` , where ` a ` sends to ` f ` and ` i ` is the identity
+       natural transformation, to the other natural transformation ` b `
+       sending to ` g ` , i.e., define the morphism part ` P ` such that
+       ` ( a ( U P V ) i ) = b ` such that ` ( b `` X ) = g ` given hypotheses
+       of ~ fuco23 .  Such construction should be provable as a functor.
+
+       Given any ` P ` , it is a morphism part of a functor compatible with the
+       object part, i.e., the functor composition, i.e., the restriction of
+       ` o.func ` , iff both of the following hold.
+
+       1.  It has the same form as ~ df-fuco up to ~ fuco23 , but
+       ` ( ( B ( U P V ) A ) `` X ) ` might be mapped to a different morphism
+       in category ` E ` .  See ~ fucofulem2 for some insights.
+
+       2. ~ fuco22nat , ~ fucoid , and ~ fucoco are satisfied.
+
```

Comment:
_As an counterexample_ > As **a** counterexample
_It seems possible_ > it **is** possible